nymea-plugins-modbus.pro

```diff
@@ -1,13 +1,14 @@
 TEMPLATE = subdirs
 
 PLUGIN_DIRS = \
-    drexelundweiss          \
-    modbuscommander         \
-    mypv                    \
-    sunspec                 \
-    unipi                   \
-    wallbe                  \
-    webasto                 \
+    mtec                    \
```

Comment:
Please undo this before pushing, otherwise we cannot start building experiemtal with this pr.

Reply:
Done, sorry!